apps/server/src/modules/oauth/uc/oauth.uc.spec.ts

```diff
@@ -65,37 +61,83 @@ describe('OAuthUc', () => {
 				},
 				{
 					provide: OAuthService,
-					useValue: {
-						processOAuth() {},
-					},
+					useValue: createMock<OAuthService>(),
 				},
 			],
 		}).compile();
 
-		service = await module.resolve<OauthUc>(OauthUc);
-		oauthService = await module.resolve<OAuthService>(OAuthService);
+		oauthService = await module.get(OAuthService);
```

Comment:
the module should be created `beforeAll`

Reply:
done